test/e2e/common/autoscaling_utils.go

```diff
@@ -24,7 +24,7 @@ import (
 	"time"
 
 	autoscalingv1 "k8s.io/api/autoscaling/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
no need.

Reply:
This won't be a big problem IMO, I ran the script `./hack/update-import-alias.sh` which introduced in https://github.com/kubernetes/kubernetes/pull/78836 and it doesn't update this.